lib/bitreserve.rb

```diff
@@ -5,18 +5,26 @@
 Dotenv.load
 
 require 'bitreserve/version'
+require 'bitreserve/options'
 require 'bitreserve/client'
 
 module Bitreserve
-  API_BASE = 'https://api.bitreserve.org'
   ROOT_PATH = File.dirname(__FILE__)
 
-  @api_base = API_BASE
+  @api_base = Options::API_BASE
   @api_version = 0
   @logger = Logger.new(STDOUT)
 
   class << self
     attr_accessor :api_base, :api_version, :logger
+
+    def sandbox=(value)
+      self.api_base = if value
```

Comment:
I don't dislike the way it is know, but I wouldn't oppose a change if the rest of the team prefers it

The only requirement I have is that vim's auto-indent should be changed as well, which I don't know how to do

PS: this is getting offtopic. I'll create an issue on the guides to discuss it